parsec/_parsec_pyi/protocol/__init__.pyi

```diff
@@ -0,0 +1,391 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
```

Comment:
It's required to re-export everything in the `__init__.pyi` ?

Reply:
You're right, it's alredy exported by the `import` statements. I fixed it